_posts/2018-10-12-strimzi-0.8.0-release.md

```diff
@@ -0,0 +1,164 @@
+---
+layout: post
+title:  "Strimzi 0.8.0 released!"
+date: 2018-10-12
+author: paolo_patierno
+---
+
+We are delighted to announce the new Strimzi 0.8.0 release with many awesome new features!
+
+<!--more-->
+
+# Access your Kafka cluster from outside—without TLS
+
+The previous release added the ability to access your Kafka cluster from the outside.
+So, Kafka clients could exchange messages even if they were not running on the same Kubernetes or OpenShift cluster as the Kafka deployment.
```

Comment:
The _So, _ seems a bit strange to me TBH ... but feel free to leave it since my English is for sure not the best.